code/src/java/pcgen/core/GameMode.java

```diff
@@ -2307,7 +2306,7 @@ public void resolveInto(AbstractReferenceContext referenceContext)
 		}
 	}
 
-	private AbstractReferenceContext getRefContext()
+	private static AbstractReferenceContext getRefContext()
```

Comment:
FWIW, the whole "method should be static because it doesn't use a field" thing is really one of those things where tools say "it can be" and I say "so what" - just because it can be doesn't mean it should be... since this is not something we've had in code standards this isn't really high on my list of cleanup I think adds value.